CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change log
 
+## 0.1.3
```

Comment:
While I appreciate the edit for 0.1.2, it should be a separate PR, and the changelog should only be updated when a new release is actually made.